ext/TensorKitChainRulesCoreExt.jl

```diff
@@ -591,9 +589,8 @@ function lq_pullback!(ΔA::AbstractMatrix, L::AbstractMatrix, Q::AbstractMatrix,
                       atol::Real=0,
                       rtol::Real=atol > 0 ? 0 : eps(real(eltype(L)))^(3 / 4))
     Ld = view(L, diagind(L))
-    p = let tol = atol > 0 ? atol : rtol * maximum(abs, Ld)
-        findlast(x -> abs(x) >= tol, Ld)
-    end
+    tol = atol > 0 ? atol : rtol * maximum(abs, Ld)
+    p = findlast(x -> abs(x) >= tol, Ld)
```

Comment:
Same here. Also, if you noticed a similar pattern in other functions like svd pullback, feel free to also update accordingly :-)

Reply:
The svd pullback already handled it this way, so no need for further changes as far as I can tell. It seems things that look "exactly the same" to my untrained eye can still be quite different :)